pages/common/fakedata.md

```diff
@@ -0,0 +1,24 @@
+# fakedata
+
+> Generate fake data using a large variety of generators.
+> More information: <https://github.com/lucapette/fakedata>.
+
+- List all valid generators:
+
+`fakedata --generators`
+
+- Generate data using one or more generators:
+
+`fakedata {{generator1}} {{generator2}}`
+
+- Generate data with a specific output format:
+
+`fakedata --format {{csv|tab|sql}} {{generator}}`
+
+- Generate a given number of data items (defaults to 10):
+
+`fakedata --limit {{n}} {{generator}}`
+
+- Generate data using a custom output template (the first letter of generator names must be capitalized):
+
+`echo "{{\{\{Generator\}\}}}" | fakedata`
```

Comment:
Oh wow, this is a very awkward situation here. We don't actually define any standard way to escape this in the [client spec](https://github.com/tldr-pages/tldr/blob/master/CLIENT-SPECIFICATION.md#page-structure). This is the only thing we mention:

> Although this specification is about the interface that clients must provide, it is also worth noting that pages are written in standard CommonMark, which the exception of the non-standard `{{` and `}}` syntax, which surrounds values in an example that users may edit. Clients MUST NOT break if the page format is changed within the CommonMark specification.

So the escaping with the backward slashes won't be handled correctly here. What happens if you remove the backwards slashes? How do clients handle that? I would assume that since the syntax can't be nested they would display it?

I think a client spec update is in order to properly define our non-standard token syntax.

Reply:
Also @sbrl some clients just blindly replace `{{` and `}}` with ANSI codes/HTML to highlight, e.g. [hidroh/tldroid](https://github.com/hidroh/tldroid/blob/b33394e48ffcabb39d99ceb7c69fd56322ca8a68/app/src/main/kotlin/io/github/hidroh/tldroid/MarkdownProcessor.kt#L30-L32) (though that client it no longer maintained and we're going to remove it soon I think, #4044)